akka-stream/src/main/scala/akka/stream/io/SslTls.scala

```diff
@@ -380,6 +380,10 @@ sealed trait SslTlsOutbound
  *  - `enabledProtocols` will be passed to `SSLEngine::setEnabledProtocols()`
  *  - `clientAuth` will be passed to `SSLEngine::setWantClientAuth()` or `SSLEngine.setNeedClientAuth()`, respectively
  *  - `sslParameters` will be passed to `SSLEngine::setSSLParameters()`
+ *
+ * Please note that passing `clientAuth = None` means that no change is done
```

Comment:
I was moving this today to ssl-config and there I made "don't change the JDK defaults" `ClientAuth.Default`. Note: the defaults are `false` on all possible flags, which I documented there. https://github.com/typesafehub/ssl-config/pull/29/files#diff-f7685010387148dd18233c93cbabf44dR7